azext_iot/_params.py

```diff
@@ -4,6 +4,10 @@
 # Licensed under the MIT License. See License.txt in the project root for license information.
 # --------------------------------------------------------------------------------------------
 
+"""
+CLI parameter definations.
```

Comment:
Small typo here.

Reply:
fixed in commit: https://github.com/Azure/azure-iot-cli-extension/pull/4/commits/ab5b4b6c8effbd8bd7253254ca28e8aaedca1d07